app/scripts/GenomePositionSearchBox.jsx

```diff
@@ -795,7 +795,7 @@ class GenomePositionSearchBox extends React.Component {
             className={styles['genome-position-search-bar-button']}
             id={this.uid}
             onChange={this.handleAssemblySelectEvt.bind(this)}
-            title={
+            value={
               this.state.selectedAssembly
                 ? this.state.selectedAssembly
                 : '(none)'
```

Comment:
Should this have the same logic as below?

`this.state.selectedAssembly ?? this.positionText ?? '(none)'`